site/profile/manifests/slurm.pp

```diff
@@ -108,22 +110,17 @@
   }
 
   $slurm_path = @(END)
-# Add Slurm custom paths for local users
-if [[ $UID -lt 10000 ]]; then
+<% if $force_slurm_in_path { %>if [[ $UID -lt <%= $uid_max %> ]]; then<% } %>
```

Comment:
This logic is the wrong way around it should be inverted

Reply:
You are right, I got that wrong.
It should be :
```<% if ! $force_slurm_in_path { %>if [[ $UID -lt <%= $uid_max %> ]]; then<% } %>
```